dashboard/server.py

```
@@ -1330,6 +1345,35 @@ def get_run_profitability(run_id: str | None = None):
+    from core_brain.order_manager import account_sweep
+    db_path = str(resolve_db_path(_ACTIVE_DB_OVERRIDE))
+    try:
+        res = account_sweep(quiet=True, db_path=db_path)
+        if isinstance(res, dict) and res.get("account_value_usd") is not None:
+            val = float(res["account_value_usd"])
+            procs_file = runtime_file("processes.json", root=LIVE_ROOT)
+            saved = {}
+            if procs_file.exists():
+                try:
+                    saved = json.loads(procs_file.read_text(encoding="utf-8"))
+                except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)